Unable to run the app in android

**Describe the bug**
A clear and concise description of what the bug is.
The project is not running in windows with android 

**To Reproduce**
Steps to reproduce the behavior:
1. Clone the app
2. Run the app
You will get the error

**Expected behavior**
Should run the app properly

### Environment
* React Native Starter version: 1.0.5
* React Native version: 0.57.2
* Platforms
  * [ ] iOS
  * [x ] Android
* Device Info
  * [x ] Physical Device in Debug
  * [ ] Physical Device in Release
  * [ ] Simulator in Debug
  * [ ] Simulator in Release


### Additional context

Here is the exception that I am getting in android studio

```
Failed to resolve: support-fragment
Open File


Failed to resolve: support-core-ui
Open File


Failed to resolve: common
Open File


Failed to resolve: play-services-base
Open File


Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html


Failed to resolve: support-fragment
Open File


Failed to resolve: support-core-ui
Open File


Failed to resolve: common
Open File


Failed to resolve: play-services-base
Open File


Failed to resolve: support-fragment
Open File


Failed to resolve: support-core-ui
Open File


Failed to resolve: runtime
Open File


Failed to resolve: common
Open File


Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html


Failed to resolve: support-fragment
Open File


Failed to resolve: support-core-ui
Open File


Failed to resolve: common
Open File


Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html


Failed to resolve: support-fragment
Open File


Failed to resolve: support-core-ui
Open File


Failed to resolve: common
Open File


Failed to resolve: support-fragment
Open File


Failed to resolve: support-core-ui
Open File


Failed to resolve: common
Open File


Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html


Failed to resolve: support-fragment
Open File


Failed to resolve: support-core-ui
Open File


Failed to resolve: common
Open File


FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':react-native-code-push:debugCompileClasspath'.
> Could not find support-fragment.aar (com.android.support:support-fragment:27.1.1).
  Searched in the following locations:
      https://jcenter.bintray.com/com/android/support/support-fragment/27.1.1/support-fragment-27.1.1.aar
> Could not find support-core-ui.aar (com.android.support:support-core-ui:27.1.1).
  Searched in the following locations:
      https://jcenter.bintray.com/com/android/support/support-core-ui/27.1.1/support-core-ui-27.1.1.aar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 23s
14 actionable tasks: 1 executed, 13 up-to-date
```

Please suggest me some solution for this

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -27,10 +27,10 @@ subprojects { subproject ->
 
 buildscript {
     repositories {
+        google()
         jcenter()
         mavenLocal()
         mavenCentral()
-        google()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.2'
@@ -44,6 +44,7 @@ buildscript {
 
 allprojects {
     repositories {
+        google()
         mavenLocal()
         mavenCentral()
         jcenter()

